docs/blog/articles/2021-10-11-KServe-0.7-release.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "The KServe 0.7 release ensures a smooth transition from KFserving to KServe"
+date: 2020-10-11
+description: "KServe 0.7 Release Blog Post"
+type: "blog"
+---
+
+### Authors
+ **Dan Sun**,**Animesh Singh**,**Vedant Padwal** on behalf of the KServe Community
+
+### **What's Changed?**
+
+- #### [Migration job for existing KFServing installation](https://kserve.github.io/website/admin/migration)
+    This helps to migrate existing inference services running in the cluster from KFServing to KServe without downtime. 
+    The script installs `KServe` from 0.7 release and deletes your `KFServing` installation after migrating the inference services from `serving.kubeflow.org` to `serving.kserve.io`.
```

Comment:
Do not need to repeat the docs here as you already provided the link, I think the migration instruction still needs to be tweaked.